scripts/rebase-fork.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+if [[ -z $1 ]]; then
+    echo "Please specify the pull request number you want to rebase to a local branch."
+    echo "Usage: ./scripts/rebase-fork.sh <pr number>"
+    echo ""
+    echo "Example: ./scripts/rebase-fork.sh 557"
+    exit 1
+fi
+
+PR_NUMBER=$1
+
+declare -a req_tools=("gh" "git" "jq")
+for tool in "${req_tools[@]}"; do
+  if ! command -v "$tool" > /dev/null; then
+    echo "It looks like '${tool}' is not installed; please install it and run this script again."
+    exit 1
+  fi
+done
+
+# Check if the PR specified is a valid number
+re='^[0-9]+$'
+if ! [[ $PR_NUMBER =~ $re ]] ; then
+   echo "The PR you specify must be a valid integer number." >&2; exit 1
+fi
+
+# Check if the specified PR exists
+gh pr view $PR_NUMBER &> /dev/null
+if [[ ! $? -eq 0 ]]; then
+    echo "Pull request #$PR_NUMBER not found."
+    exit 1
+fi
+
+# Checkout the fork PR locally
+gh pr checkout $PR_NUMBER
+
+# Grab the PR title and branch name
+FORK_PR_TITLE=$(gh pr view $PR_NUMBER --json title | jq '.title' | tr -d '"')
+FORK_PR_BRANCH=$(gh pr view $PR_NUMBER --json headRefName | jq '.headRefName')
+
+# Name of the local branch that will be pushed upstream
+LOCAL_BRANCH="local/$(echo $FORK_PR_BRANCH | tr -d '"')"
+
+# Fetch the PR body and write to local markdown file
+gh pr view $PR_NUMBER --json body | jq -r '.body' > ${PR_NUMBER}.md
+
+git checkout -b $LOCAL_BRANCH
+git commit --allow-empty -m "Rebased $FORK_PR_BRANCH onto a local branch"
+git push -u origin $LOCAL_BRANCH
+
+# Finally we can automagically open a new PR using the fork PR's original title
+# and description
+gh pr create --title="${FORK_PR_TITLE}" --body-file=$PR_NUMBER.md
```

Comment:
💅 I lean towards using curly braces more often than not in scripts:

```suggestion
gh pr create --title="${FORK_PR_TITLE}" --body-file=${PR_NUMBER}.md
```